manual/core/integration/README.md

```diff
@@ -0,0 +1,287 @@
+## Integration
+
+This page contains various information on how to integrate the driver in your application.
+
+### Minimal project structure
+
+We publish the driver to [Maven central][central_oss]. Most modern build tools can download the
+dependency automatically.
+
+#### Maven
+
+Create the following 4 files:
```

Comment:
Maybe we should consider creating and publish maven archetype of our own to kickstart a project which creates all the relevant files via `mvn archetype:generate`.

Reply:
Maybe, it might be cool to have generation options for the different setups: shaded or not, pick a logging framework, etc. On the other hand, that's another thing to maintain....

Feel free to open a ticket if you want to pursue the idea.